src/const.js

```diff
@@ -0,0 +1,2 @@
+export const digits = ["9", "8", "7", "6", "5", "4", "3", "2", "1", "0"];
+export const operators = ["/", "X", "-", "+", "="];
```

Comment:
수정 불가능하도록 고치거나 상수라는 것을 명시적으로 표현하는게 어떨까요?

Reply:
```suggestion
export const DIGIT_NUMBERS = Object.freeze(["9", "8", "7", "6", "5", "4", "3", "2", "1", "0"]);
export const OPERATORS = Object.freeze(["/", "X", "-", "+", "="]);

```

수정이 불가능하도록 Object.freeze 를 사용하고, 변수명만으로 상수임을 알 수 있게끔 대문자로 변경해봤습니다!
TypeScript를 사용했다면 간단히 `as const`로 제어할 수 있는 부분인데 이런 부분에서 필요성을 느끼네요!

Reply:
오 공부를 열심히 하셨네요 👍